### PR TITLE
Duel wielding nerf

### PR DESCRIPTION
**Pull request by David-Stormwell**
## About The Pull Request

The long awaited PR.
This nerfs duel wielding quite some via increasing scatter and lowering accuracy.
Also, changes up the old duel wielding code a tad, to make it cleaner and so you can add specific values for a gun if required. 
 
## Why It's Good For The Game

It's required due to the current TP-14 spam sadly, but in the end at least guns won't have to be balanced around duel wielding and instead simply for 1 hand or wielded styles only.

## Changelog
:cl:
balance: Made Akimbo/Duel wielding less viable on the TP-14
refactor: Adds vars for scatter on guns, you may now add specific values on a gun if required. 
/:cl:

### CODE REVIEW

**Comment by David-Stormwell** [comment]
need to change something don't look right now at the code æ

**Comment by BraveMole** [comment]
why nerfing rifle akimbo? it's bad already

**Comment by David-Stormwell** [comment]
> why nerfing rifle akimbo? it's bad already

I'll do some testing.
Just keep in mind this is supposed to nerf T-90 and other sub machine guns from being used to duel wield.

**Comment by BraveMole** [comment]
> > why nerfing rifle akimbo? it's bad already
> 
> I'll do some testing.
> Just keep in mind this is supposed to nerf T-90 and other sub machine guns from being used to duel wield.

I don't know, i think T90 akimbo is already bad? I can't barely shoot any xeno that are more than 3 tiles away, and using this loadout you'll probably FF like crazy

**Comment by davidchan13** [comment]
Just nerf TP14 directly, nothing else works akimbo if you aren't already point blank range.

**Comment by Georgeur** [comment]
tp23 is in most cases an upgrade to the tp14
it is not an issue once #6570  is merged because they wont have an effective mag size of 483
whilst I'm totally for a tp14 change, a 33% reduction in sunder would help to properly balance it with the tp23 (same sunder per mag that way) 
this is a crude implementation

**Comment by Cowbin** [comment]
If you wish to nerf the akimbo, just do so directly, there aren't really any other balance issues with dual wielding in general

**Comment by David-Stormwell** [comment]
> If you wish to nerf the akimbo, just do so directly, there aren't really any other balance issues with dual wielding in general

This is a direct Akimbo nerf?
All this does is make akimbo less viable.
I'm going to switch some numbers around now that I have a good look at how it works, most likely decrease the accuracy nerf so if you're at least at PB range it'll still be ok.
Combined with Harry's TP 14 Sunder nerf it'll be worse option, but if you still want to use it it'd be ok against T1s and the such.

File: code/modules/projectiles/gun_system.dm [hunk 1]
```diff
@@ -938,11 +938,11 @@ and you're good to go.
...
 		if(gun_skill_category == GUN_SKILL_PISTOLS)
-			gun_accuracy_mult = max(0.1, gun_accuracy_mult - 0.1*rand(1,2))
-			gun_scatter += 10*rand(1,3)
+			gun_accuracy_mult = max(0.1, gun_accuracy_mult - 0.5*rand(1,2))
+			gun_scatter += 15*rand(1,3)
 		else
-			gun_accuracy_mult = max(0.1, gun_accuracy_mult - 0.1*rand(2,4))
-			gun_scatter += 10*rand(3,5)
+			gun_accuracy_mult = max(0.1, gun_accuracy_mult - 0.5*rand(2,4))
+			gun_scatter += 15*rand(3,5)
```
Comment (SplinterGP) [inline code/modules/projectiles/gun_system.dm:945]: non pistols cant akimbo already and youre increasing a value by atleast 5x
```suggestion
			gun_accuracy_mult = max(0.1, gun_accuracy_mult - 0.1*rand(2,4))
			gun_scatter += 10*rand(3,5)
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Comment by Georgeur** [comment]
> > If you wish to nerf the akimbo, just do so directly, there aren't really any other balance issues with dual wielding in general
> 
> This is a direct Akimbo nerf?
> All this does is make akimbo less viable.
> I'm going to switch some numbers around now that I have a good look at how it works, most likely decrease the accuracy nerf so if you're at least at PB range it'll still be ok.
> Combined with Harry's TP 14 Sunder nerf it'll be worse option, but if you still want to use it it'd be ok against T1s and the such.

go for spread and not inaccuracy, more concrete stats that way, also easier to make a damage table, but like I did say before, this is a crude implementation
especially considering your own #6570 which fixes most the problems

**Comment by Dolth** [comment]
TP14 dual wielding is currently OP.
Dual wielding for most weapons is not OP.
You're nerfing dual wielding therefor not only TP14 dual wielding but everything else.

Kinda missed the target don't you think?

**Comment by David-Stormwell** [comment]
> 
> 
> TP14 dual wielding is currently OP.
> Dual wielding for most weapons is not OP.
> You're nerfing dual wielding therefor not only TP14 dual wielding but everything else.
> 
> Kinda missed the target don't you think?

I'm removing the nerf+ in my next commit for non pistol weapons.
Otherwise TP-14 is the only gun which should be duel-wielded if at all in the first place.

**Comment by SplinterGP** [comment]
> 
> 
> > TP14 dual wielding is currently OP.
> > Dual wielding for most weapons is not OP.
> > You're nerfing dual wielding therefor not only TP14 dual wielding but everything else.
> > Kinda missed the target don't you think?
> 
> I'm removing the nerf+ in my next commit for non pistol weapons.
> Otherwise TP-14 is the only gun which should be duel-wielded if at all in the first place.

TP-14, Tp-23, TP-44, 88mod, Tp-17. other guns should be dual wieldable in specific cases, like using two LMG's with bipod, even though with some considerable scatter and maybe recoil

File: code/modules/projectiles/gun_system.dm [hunk 2]
```diff
@@ -938,11 +938,11 @@ and you're good to go.
 
 	if(dual_wield) //akimbo firing gives terrible accuracy
 		if(gun_skill_category == GUN_SKILL_PISTOLS)
-			gun_accuracy_mult = max(0.1, gun_accuracy_mult - 0.1*rand(1,2))
-			gun_scatter += 10*rand(1,3)
+			gun_accuracy_mult = max(0.1, gun_accuracy_mult - 0.5*rand(1,2))
```
Comment (SplinterGP) [inline code/modules/projectiles/gun_system.dm:941]: accuracy is a wack mechanic and shoulndt be mostly used for anything, most stuff should miss if it hits, use scatter
```suggestion
			gun_accuracy_mult = max(0.1, gun_accuracy_mult - 0.1*rand(1,2))
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/gun_system.dm (hunk 2 above)
Comment (David-Stormwell) [reply code/modules/projectiles/gun_system.dm]: Decrceased the accuracy nerf heavily in the latest commit.
It still has 10 max, but that's quite negligible.

**Review by SplinterGP — APPROVED** [review]
seems reasonable now, good to go

**Comment by David-Stormwell** [comment]
Finished it, only problem is that in my testing it's still pretty good even base wise.
Sadly gonna need https://github.com/tgstation/TerraGov-Marine-Corps/pull/6585 to go along with it so we can finish it.

**Comment by David-Stormwell** [comment]
when someone asks for a refactor

**Comment by David-Stormwell** [comment]
And finished.
That's one thing out of the way.

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**